<?xml version="1.0" encoding="UTF-8" ?>
<testsuites disabled="0" errors="0" failures="1" tests="2" time="11.207">
 <testsuite name="demo calculator test" timestamp="2021-02-27T17:17:04" hostname="localhost" time="11.207" errors="0" tests="2" skipped="0" disabled="0" failures="1">
  <testcase classname="demo calculator test" name="addition test" time="6.513" />
  <testcase classname="demo calculator test" name="subscription test" time="4.692">
   <failure type="toBe" message="Expected &apos;Go!&apos; to be &apos;1&apos;."><![CDATA[Error: Failed expectation
    at UserContext.<anonymous> (/Users/BharathG/Documents/Automation/Protractor/tests/calculator.js:25:32)
    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7]]>
   </failure>
  </testcase>
 </testsuite>
</testsuites>